hack/docker/setup.sh

```diff
@@ -82,6 +82,9 @@ build_docker() {
   cat >Dockerfile <<EOL
 FROM alpine:3.8
 
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN addgroup -S stash && adduser -S -G stash stash
```

Comment:
We should use a number as the user id instead of username.

Reply:
Used numeric id